src/gfx/gizmos/translation_gizmo.hpp

```diff
@@ -6,10 +6,15 @@
 
 namespace gfx {
 namespace gizmos {
+// Note: While only a single handle of the TranslationGizmo may be selected at any point in time,
```

Comment:
Shouldn't occur if they are used within the same gizmo context (see other comment in *.edn)